okio/src/jvmMain/kotlin/okio/-Platform.kt

```diff
@@ -52,3 +52,5 @@ actual typealias EOFException = java.io.EOFException
 actual typealias FileNotFoundException = java.io.FileNotFoundException
 
 actual typealias Closeable = java.io.Closeable
+
+actual typealias JvmDefault = kotlin.jvm.JvmDefault
```

Comment:
It's strange that they don't have it in the common stdlib artifact, unlike some other annotations (https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.jvm/). Should we file an issue (unless someone already did it)?

Reply:
Yep.

Reply:
The better approach is to switch to `-Xjvm-default=all` so that you don't need to annotate anything.

Reply:
Oh yes, that's what I want.